tests/runtime/ray/test_integration.py

```diff
@@ -698,6 +698,9 @@ class Test3rdPartyLibraries:
     # likely to be different from the one we're using for development. The SDK shows a
     # warning when deserializing a workflow def like this.
     @pytest.mark.filterwarnings("ignore::orquestra.sdk.exceptions.VersionMismatch")
+    @pytest.mark.skipif(
+        sys.version_info < (3, 11), reason="Pickle internals changed in Python 3.11"
```

Comment:
If there's inter-version mismatch I think we should focus on the newest one.
* Have you considered updating the test data from `3.11` and skipping the test if running on `3.10` and below?
* If you agree, but that's too much of a task for now, could you add a separate ticket for it?

Reply:
That's what this does

Reply:
Holy moly you're right